src/components/SectionHeader/index.jsx

```diff
@@ -17,10 +22,12 @@ const styles = () => ({
 });
 
 const SectionHeader = ({ classes, title, subtitle }) => (
```

Comment:
Is the SectionHeader used in places other than <Section>?  This should be inlined, or made private.